dev/ToastNotifications/ToastNotificationManager.cpp

```diff
@@ -71,7 +77,10 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
             UnRegisterComServer(storedComActivatorString);
             UnRegisterAppIdentifierFromRegistry();
 
-            // TODO: Remove ToastGuid reference from LRP
+            wil::unique_cotaskmem_string processName;
+            THROW_IF_FAILED(GetCurrentProcessPath(processName));
+            auto notificationPlatform{ PushNotificationHelpers::GetNotificationPlatform() };
```

Comment:
Where is the platform->AddToastREgistration() call in ToastNotificationManager? Its kinda strange to see Remove without a corresponding Add

Reply:
The call happens in PushNotificationManager::CreateChannelAsync line 210 when a channel is successfully created. The mapping doesn't happen until a sink is registered.

Reply:
Got it. I hope we are using the same appId generated in CreateChannelAsync when we call into REgisterToastActivator()

Reply:
The key for this mapping is processName->appId(generated GUID/AUMID). It is the same key in LRP.

Reply:
We can call AddToastRegistration in RegisterToastActivator, but there wouldn't be a sink yet. I guess we have to call it here as well. Should we not throw if there is no sink yet as suggested in another comment?

Reply:
Do not throw there. Just return S_OK if there is no long running sink

Reply:
And yes. Please call AddToastREgistrationMapping in REgisterToastActivator. There is no other way to fix this.